distribution/docs/src/main/resources/content/_running/available-console-commands.adoc

```diff
@@ -0,0 +1,50 @@
+:title: Available Console Commands
+:type: maintaining
+:status: published
+:parent: Console Commands
+:summary: Types of console commands available.
+:order: 02
+
+== {title}
+
+${branding} provides several types of console commands for administrators to use to perform tasks.
+Additionally, many default Karaf commands are also enabled.
+For more information about Karaf commands, see the http://karaf.apache.org/documentation.html[Karaf documentation].
```

Comment:
There are more than just the default Karaf commands. Some libraries that we have installed also add some commands, for example decanter.

```Many console commands are available, including DFF commands and the core Karaf console commands. For more information about these core Karaf commands and using the console, see ...```

Also, this should link to the current version of Karaf because the available commands do change from version to version.

Reply:
Updated to use your sentence. 

Changed the link as well to link directly to the Commands docs

Reply:
I mean the link should point to the documentation for the current version of karaf in DDF, 4.1.2. If DDF is not using the most recent karaf release, this documentation at that http://karaf.apache.org/documentation.html link might not include the commands that are in DDF.